ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -69,16 +82,97 @@ open class CommandBar: UIView {
         updateShadow()
     }
 
+    /// Scrollable items shown in the center of the CommandBar
+    public var itemGroups: [CommandBarItemGroup] {
+        didSet {
+            for view in stackView.arrangedSubviews {
+                view.removeFromSuperview()
+            }
+
+            _buttonGroupViews = nil
+            for view in buttonGroupViews {
+                stackView.addArrangedSubview(view)
+            }
+        }
+    }
+
+    /// Items pinned to the leading end of the CommandBar
+    public var leadingItemGroups: [CommandBarItemGroup] {
+        didSet {
+            for view in leadingStackView.arrangedSubviews {
+                view.removeFromSuperview()
+            }
+
+            _leadingButtonGroupViews = nil
+            for view in leadingButtonGroupViews {
+                leadingStackView.addArrangedSubview(view)
+            }
+        }
+    }
+
+    /// Items pinned to the trailing end of the CommandBar
+    public var trailingItemGroups: [CommandBarItemGroup] {
+        didSet {
+            for view in trailingStackView.arrangedSubviews {
+                view.removeFromSuperview()
+            }
+
+            _trailingButtonGroupViews = nil
+            for view in trailingButtonGroupViews {
+                trailingStackView.addArrangedSubview(view)
+            }
+        }
+    }
+
     // MARK: - Private properties
 
-    private let itemGroups: [CommandBarItemGroup]
+    private var _itemsToButtonsMap: [CommandBarItem: CommandBarButton]?
```

Comment:
FYI, we don't do leading underscores for internal variable names!

Reply:
Oh, and it's the same name as itemsToButtonsMap below, with only the underscore as the difference... The intent of the two variables should be made more distinct and clear.

Reply:
Per offline discussions, _itemsToButtonsMap does not need to have an underscore in the naming and the two private itemsToButtonsMap properties can be combined into one.

Reply:
Introduced the CommandBarCommandGroupsStackView that encapsulates most of the duplicate logic I introduced with the first commit. This work removes the use of the ivars with the prefixed variable names.